sdk/python/requirements.in

```diff
@@ -22,7 +22,7 @@ kfp-pipeline-spec>=0.1.16,<0.2.0
 # Update the lower version when kfp sdk depends on new apis/fields in
 # kfp-server-api.
 # Note, please also update ./requirements.in
-kfp-server-api>=2.0.0a0,<3.0.0
+kfp-server-api>=2.0.0a7,<3.0.0
```

Comment:
JFYI, we're in progress updating APIs for v2, once that is done (ETA: end of the month or early January), the next release would be the first beta.